bundle install --local cannot skip groups, use environment variables instead

--- a/Gemfile
+++ b/Gemfile
@@ -65,13 +65,17 @@
 
 # Database
 
-group :mysql, optional: true do
-  gem "mysql2", "~> 0.5.2"
-end
-group :postgresql, optional: true do
-  gem "pg",     "~> 1.1",">= 1.1.3"
-end
+#group :mysql, optional: true do
+#  gem "mysql2", "~> 0.5.2"
+#end
+#group :postgresql, optional: true do
+#  gem "pg",     "~> 1.1",">= 1.1.3"
+#end
 
+ENV["DB"] ||= "mysql"
+
+gem "mysql2", "~> 0.5.2" if ENV["DB"] == "all" || ENV["DB"] == "mysql"
+gem "pg", "~> 1.1",">= 1.1.3" if ENV["DB"] == "all" || ENV["DB"] == "postgres"
 
 gem "activerecord-import", "~> 0.27.0"
 
